Sources/GitBuddyCore/Changelog/ChangelogProducer.swift

```diff
@@ -38,8 +38,20 @@ final class ChangelogProducer: URLSessionInjectable {
     let from: Date
     let to: Date
     let project: GITProject
+    let useGitHubReleaseNotes: Bool
+    let tagName: String?
+    let targetCommitish: String?
+    let previousTagName: String?
 
-    init(since: Since = .latestTag, to: Date = Date(), baseBranch: Branch?) throws {
+    init(
+        since: Since = .latestTag,
+        to: Date = Date(),
+        baseBranch: Branch?,
+        useGitHubReleaseNotes: Bool = false,
+        tagName: String? = nil,
+        targetCommitish: String? = nil,
```

Comment:
What is a `commitish` referring to here? Can we add a bit of documentation as to what kind of values are expected here?

Reply:
These parameters are all duplicates of the ChangelogCommand, but it doesn't hurt to duplicate the documentation as well 👌